resolwe_bio/processes/chip_seq/macs2.yml

```diff
@@ -32,129 +31,257 @@
+            transcription factor is 200 bp, and you want to bypass the model building by MACS,
+            this parameter can be set as 200. This option is only valid when --nomodel is set or
+            when MACS fails to build model and --fix-bimodal is on.
         - name: shift
           label: Shift
           type: basic:integer
           required: false
           description: >
-            Note, this is NOT the legacy --shiftsize option which is replaced by --extsize! You can set an arbitrary shift in bp here. Please Use discretion while setting it other than default value (0). When --nomodel is set, MACS will use this value to move cutting ends (5') then apply --extsize from 5' to 3' direction to extend them to fragments. When this value is negative, ends will be moved toward 3'->5' direction, otherwise 5'->3' direction. Recommended to keep it as default 0 for ChIP-Seq datasets, or -1 * half of EXTSIZE together with --extsize option for detecting enriched cutting loci such as certain DNAseI-Seq datasets. Note, you can't set values other than 0 if format is BAMPE for paired-end data. Default is 0.
+            Note, this is NOT the legacy --shiftsize option which is replaced by --extsize! You
+            can set an arbitrary shift in bp here. Please Use discretion while setting it other
+            than default value (0). When --nomodel is set, MACS will use this value to move
+            cutting ends (5') then apply --extsize from 5' to 3' direction to extend them to
+            fragments. When this value is negative, ends will be moved toward 3'->5' direction,
+            otherwise 5'->3' direction. Recommended to keep it as default 0 for ChIP-Seq datasets,
+            or -1 * half of EXTSIZE together with --extsize option for detecting enriched cutting
+            loci such as certain DNAseI-Seq datasets. Note, you can't set values other than 0 if
+            format is BAMPE for paired-end data. Default is 0.
         - name: band_width
           label: Band width
           type: basic:integer
-          placeholder: '200'
           required: false
           description: >
-            The band width which is used to scan the genome ONLY for model building. You can set this parameter as the sonication fragment size expected from wet experiment. The previous side effect on the peak detection process has been removed. So this parameter only affects the model building.
+            The band width which is used to scan the genome ONLY for model building. You can set
+            this parameter as the sonication fragment size expected from wet experiment. The
+            previous side effect on the peak detection process has been removed. So this parameter
+            only affects the model building.
         - name: nolambda
           label: Use backgroud lambda as local lambda
           type: basic:boolean
           default: false
+          required: false
```

Comment:
The reason for `required:false` on `basic:boolean` inputs is purely aesthetic. I didn't like there is a `*` next to them. Since the bool input's default value is false (frontend) there is no need for it to be required - it makes no sense.